### PR TITLE
V1.2.1 Release Prep

### PR DESCRIPTION
**Pull request by rakuy0**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,25 @@
 Vivisect Changelog
 ******************
 
+V1.2.1 - 2024-07-24
+===================
+Features
+--------
+- Update to large resource class in CI config.
```
Comment (invisig0th) [inline CHANGELOG.rst:9]: nit, but this should probably be moved down to fixes since it's not really a viv feature and this is a patch level release :)

**Review by invisig0th — CHANGES_REQUESTED** [review]

File: CHANGELOG.rst (hunk 1 above)
Comment (rakuy0) [reply CHANGELOG.rst]: Feature felt a little closer than fix, but it's not really either of them I guess. So I just renamed it devops and added a label for that.

**Comment by williballenthin** [comment]
i've got a few pending bug fixes i can have over tomorrow, if you'll wait up a little.

**Comment by rakuy0** [comment]
Yea I can hold things

**Review by ma77carpenter — APPROVED** [review]

**Comment by williballenthin** [comment]
#659

**Comment by williballenthin** [comment]
would you please include 3.8 until it is EOL?

**Comment by rakuy0** [comment]
Sure, just with the caveat that iirc we've never ran 3.8 in any of our CI, so it's a bit of a "run at your own risk" scenario.

**Comment by williballenthin** [comment]
it runs on our CI for capa so i'd say we have fair coverage for its performance and regressions.

**Comment by rakuy0** [comment]
Ah, that's fair. I don't keep strong track of other CI setups. But good to know :+1:

File: pyproject.toml [hunk 2]
```diff
@@ -4,19 +4,19 @@ build-backend = 'setuptools.build_meta'
...
 name = 'vivisect'
-version = '1.2.0'
+version = '1.2.1'
 authors = [
     { name =  'The Vivisect Peeps', email = 'vivdevs@r4780y.com'}
 ]
 description = 'Pure python disassembler, debugger, emulator, and static analysis framework'
 readme = 'README.md'
-requires-python = '>=3.11'
+requires-python = '>=3.8'
```
Comment (atlas0fd00m) [inline pyproject.toml:13]: :+1:

**Review by atlas0fd00m — APPROVED** [review]
gtg

**Pull request merged**